lib/modules/manager/tox/extract.ts

```diff
@@ -0,0 +1,136 @@
+  return parmName;
+}
+
+/**
+ * Taken from the setup.cfg manager due to some edits to the dep type.
+ */
+function parseDep(line: string): PackageDependency | null {
+  const packagePattern = '[a-zA-Z0-9]|[a-zA-Z0-9][a-zA-Z0-9._-]*[a-zA-Z0-9]';
+  const extrasPattern = '(?:\\s*\\[[^\\]]+\\])?';
+
+  const rangePattern: string = RANGE_PATTERN;
+  const specifierPartPattern = `\\s*${rangePattern.replace(
+    regEx(/\?<\w+>/g),
+    '?:'
+  )}`;
+  const specifierPattern = `${specifierPartPattern}(?:\\s*,${specifierPartPattern})*`;
+  const dependencyPattern = `(${packagePattern})(${extrasPattern})(${specifierPattern})`;
+
+  const pkgRegex = regEx(`^(${packagePattern})$`);
+  const pkgValRegex = regEx(`^${dependencyPattern}$`);
+
+  const [lineNoEnvMarkers] = line.split(';').map((part) => part.trim());
+  const packageMatches =
+    pkgValRegex.exec(lineNoEnvMarkers) ?? pkgRegex.exec(lineNoEnvMarkers);
+
+  if (!packageMatches) {
+    return null;
+  }
+
+  const [, depName, , currVal] = packageMatches;
+  const currentValue = currVal?.trim();
+
+  const dep: PackageDependency = {
+    depName,
+    currentValue,
+    datasource: PypiDatasource.id,
+    depType: 'test',
+  };
+
+  if (currentValue?.startsWith('==')) {
+    dep.currentVersion = currentValue.replace(/^==\s*/, '');
+  }
+
+  return dep;
+}
+
+/**
+ * Map over the lines keeping track if the section is a test env
+ * and the parameter is deps =
+ *
+ * It is only valid to have a single dep with
+ *
+ * deps = <dep>
+ *
+ * or deps with indents like so
+ *
+ * deps =
+ *     dep1
+ *     dep2
+ *
+ * Attempting to use use npm ini does not parse deps arrays correctly.
+ */
+export function extractPackageFile(
+  content: string
+): Result<PackageFile | null> {
+  logger.trace('tox.extractPackageFile()');
+
+  let sectionName: string | null = null;
+  let paramName: string | null = null;
+  let inDeps = false;
+  let isTestEnv = false;
+
+  const deps: PackageDependency[] = [];
+  const testEnvRegex = regEx(/^(testenv)\S*\s*$/);
+
+  content
+    .split(newlineRegex)
+    .map((line) => line.replace(regEx(/#.*$/), '').trimEnd())
+    .forEach((rawLine) => {
+      let line = rawLine;
+      const newSectionName = getSectionName(line);
+
+      // Keep track of when sections change
+      if (is.string(newSectionName)) {
+        sectionName = newSectionName;
+        isTestEnv = testEnvRegex.test(sectionName);
+        inDeps = false;
+        return;
+      }
+
+      // Only parse if inside testenv and param name is deps =
+      const depsReplaceRegex = regEx(/^deps\s*=\s*/);
+      if (isTestEnv === true) {
+        paramName = getParamName(line);
+        if (is.string(paramName) && paramName === 'deps') {
+          inDeps = true;
+          // Can also have deps on this line, so just remove the preceeding
+          // deps =
+          line = line.replace(depsReplaceRegex, '');
+        } else if (is.string(paramName) && paramName !== 'deps') {
```

Comment:
The changes to `getParamName` allows here a simplification
```suggestion
        } else if (paramName !== 'deps') {
```